README.md

```diff
@@ -16,11 +16,13 @@ To use the latest version of monasca-agent, simply run
 ./create_metrics_agent_installer.sh 
 ```
 
-To use a specific version of monasca-agent, add the desired version number as an argument:
+To use a specific version of monasca-agent, add the desired version number and upper constraint file as an argument:
 ```
-./create_metrics_agent_installer.sh <version_number>
+./create_metrics_agent_installer.sh <version_number> <upper_constraint_file>
 ```
 
+You can find upper constraint file in [here](https://github.com/openstack/requirements/blob/master/upper-constraints.txt)
```

Comment:
You can find an example of an upper constraint file [here]

Reply:
Done.